models/models.go

```diff
@@ -11,4 +11,12 @@ type Project struct {
 	Tags string
 	RepoLink string
 	ComChannel string
-}
\ No newline at end of file
+}
+
+type Mentor struct{
+	gorm.Model
+	Name string
+	Email string
+	acc_tok string
+	git_han string
```

Comment:
`git_han` and `acc_tok` should start with Upper case right?
Else we can't access them.